src/components/button-toggle-group/button-toggle-group.stories.mdx

```diff
@@ -109,6 +109,33 @@ import ButtonToggleGroup from "carbon-react/lib/components/button-toggle-group";
   </Story>
 </Canvas>
 
+### With fullFlex
```

Comment:
not essential, as I know it appears in the props table at the bottom, but I'd prefer to add a very brief 1 or 2 sentences of documentation here to describe what the prop does.